requirements.txt

```
@@ -1,3 +1,4 @@
 psutil
 requests
-scikit-build-core
\ No newline at end of file
+scikit_build_core
+wheel
```

```suggestion
wheel
networkx
numpy==1.18.4
```

This pip packages are removed on this PR from the old requirements.txt, so I think should be added here.